examples/api/test/material/stepper/stepper.controls_builder.0_test.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/stepper/stepper.controls_builder.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('It should override the control builder to display custom buttons', (WidgetTester tester) async {
```

Comment:
What is "It"? Please describe the subject

Reply:
I changed the test description in [test: Change test description](https://github.com/flutter/flutter/pull/150669/commits/6da71d3cb8c72bfad6b504554ad55ad8518662a7). Is that an acceptable description?